RadarSDKTests/Resources/search_points.json

```diff
@@ -0,0 +1,62 @@
+{
+   "meta":{
+      "code":200,
+      "hasMore":false
```

Comment:
interesting, this is the first time I've seen this meta key - is it for pagination?

Reply:
Oh, sorry, search shall not return this field. I copied the JSON response from points get call (which has pagination)